### PR TITLE
Updating devtool installation

### PR DESCRIPTION
**Pull request by Laxzal**
devtool bash scripts search for mamba, and if found, will use mamba as the installation command.

### CODE REVIEW

File: devtools/install_autotst.sh [hunk 1]
```diff
@@ -2,6 +2,16 @@
 CONDA_BASE=$(conda info --base)
 source $CONDA_BASE/etc/profile.d/conda.sh
 
+
+if [ $(mamba) ] > /dev/null 2>&1; then
```
Comment (kfir4444) [inline devtools/install_autotst.sh:6]: Can you explain this line to me?

File: devtools/install_autotst.sh [hunk 2]
```diff
@@ -2,6 +2,16 @@
 CONDA_BASE=$(conda info --base)
 source $CONDA_BASE/etc/profile.d/conda.sh
 
+
+if [ $(mamba) ] > /dev/null 2>&1; then 
+	COMMAND_PKG=conda
+	echo "conda found"
+	else
+	COMMAND_PKG=mamba
+	echo "mamba found"
```
Comment (kfir4444) [inline devtools/install_autotst.sh:11]: What if the user doesn't have both?

**Review by kfir4444 — CHANGES_REQUESTED** [review]
Thanks @Laxzal!
I have some comments/questions, could you address them?

File: devtools/install_autotst.sh (hunk 1 above)
Comment (Laxzal) [reply devtools/install_autotst.sh:6]: It's just a redirection line. 
/dev/null accepts and discards all input;
2>&1 merges output from stream 2 with stream 1

So basically I am telling the system that it doesn't need to report anything in the terminal to the user

File: devtools/install_autotst.sh (hunk 2 above)
Comment (Laxzal) [reply devtools/install_autotst.sh:11]: I guess you could echo that they are missing an installation command but I think they would run into this error when it cannot install anyway. 

Further, maybe we should add to the website instructions to just tell them to install anaconda or mambaforge

File: devtools/install_sella.sh [hunk 3]
```diff
@@ -2,18 +2,46 @@
...
+touch ./etc/conda/deactivate.d/env_vars.sh
+
+#This sets up the LD_LIBRARY_PATH to include Sella_Env, but only when the environment is active
+echo 'export OLD_LD_LIBRARY_PATH=${LD_LIBRARY_PATH}' >> $CONDA_BASE/envs/sella_env/etc/conda/activate.d/env_vars.sh
+echo 'export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:'"$CONDA_BASE"'/envs/sella_env/lib' >> $CONDA_BASE/envs/sella_env/etc/conda/activate.d/env_vars.sh
+#This will reset the LD_LIBRARY_PATH back to the original LD_LIBRARY_PATH when the environment is deactivated
+echo 'export LD_LIBRARY_PATH=${OLD_LD_LIBRARY_PATH}' >> $CONDA_BASE/envs/sella_env/etc/conda/deactivate.d/env_vars.sh
+echo 'unset OLD_LD_LIBRARY_PATH' >> $CONDA_BASE/envs/sella_env/etc/conda/deactivate.d/env_vars.sh
+
+
```
Comment (kfir4444) [inline devtools/install_sella.sh:44]: Can you remove some of these empty lines?

**Review by kfir4444 — COMMENTED** [review]
Thanks @Laxzal!
One quick comment

**Review by kfir4444 — APPROVED** [review]
Thanks @Laxzal! Great job!

**Pull request merged**